addons/resource.language.en_gb/resources/strings.po

```diff
@@ -20099,7 +20108,19 @@ msgctxt "#36436"
 msgid "Select OS keyboard layout."
 msgstr ""
 
-#empty strings from id 36437 to 36441
+#. Option for setting "System -> Add-ons -> Addon update mode"
+#: system/settings/settings.xml
+msgctxt "#36437"
+msgid "Prefer trusted repositories"
```

Comment:
I think specifying the default here is important: `Prefer trusted repositories (default)`.